### PR TITLE
[Android] Add functions of autocapture to run in android Activities

### PR DESCRIPTION
**Pull request by park-sang-hyun**
[Edit] nnstreamer-jni.h : Add data structure to contain conditions for autocapture.
[Edit] nnstreamer-jni.c : Add 'NativeDeleteLineAndLabel', 'NativeInsertLineAndLabel', 'NativeGetCondition', 'NativeGetAutoCapture' functions based on jni.
[Edit] nnstreamer-ex.cpp : Implement 'NativeDeleteLineAndLabel', 'NativeInsertLineAndLabel', 'NativeGetCondition', 'NativeGetAutoCapture' functions.

When objects that meet the conditions are visible on the screen, can take picture automatically.
When in autocapture mode, can erase the label's name and line so that only objects can appear in the picture taken.

### CODE REVIEW

**Comment by Soyeong-k-1230** [comment]
LGTM!

File: android/example_app/nnstreamer-ssd-autocapture/README.md [hunk 1]
```diff
@@ -65,4 +65,5 @@ This page is for setting conditions. Users can add a condition on input informat
 
 <img src="res\screenshot\screenshot_preview.jpg" alt="image-preview" style="zoom:67%;" />
 
-After taking a picture, users can check the picture on this page. Users can save it or take a picture again.
\ No newline at end of file
+After taking a picture, users can check the picture on this page. Users can save it or take a picture again.
+>>>>>>> upstream/master
```
Comment (Joojihwan) [inline android/example_app/nnstreamer-ssd-autocapture/README.md:69]: How about delete this line

Comment (park-sang-hyun) [reply android/example_app/nnstreamer-ssd-autocapture/README.md]: thanks!

**Comment by macaron10** [comment]
LGTM!

**Comment by Joojihwan** [comment]
LGTM

**Review by anyj0527 — COMMENTED** [review]
Please rebase the branch. Only the last commit seems meaningful

**Comment by anyj0527** [comment]
@park-sang-hyun Please make this as a single commit PR

**Comment by park-sang-hyun** [comment]
Thank you for your answer Mentor... I've tried...
But I don't know how, so I'm going to get a fork again and try to pr.

**Comment by zhoonit** [comment]
> Thank you for your answer Mentor... I've tried...
> But I don't know how, so I'm going to get a fork again and try to pr.

You don't need to fork again and make another PR.

You need to do something like 
```bash
git rebase -i HEAD~3
(squash everything with resolving conflicts)
git push -f 
```

**Comment by park-sang-hyun** [comment]
I tried the base ~i HEAD~3 and it stopped
because even the native commitments were canceled. Why is that...?

**Comment by zhoonit** [comment]
What does `git status` say?

**Comment by park-sang-hyun** [comment]
When i HEAD-3 was attempted, about 10 commits were unstaged, including 'native' and 'android'

**Comment by zhoonit** [comment]
Commits cannot be 'unstaged'. I think you are having confusions with some terms. DM me in the slack :)

**Comment by park-sang-hyun** [comment]
Thank you for your advice...
Can I try it again and if there's something that doesn't work, can I capture it and ask you again...?

**Comment by zhoonit** [comment]
> Thank you for your advice...
> Can I try it again and if there's something that doesn't work, can I capture it and ask you again...?

Sure. Please read through https://git-scm.com/book/en/v2/

or at least refer to https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by park-sang-hyun** [comment]
Thank you very much!

**Pull request closed without merging**